djangocms_picture/migrations/0003_migrate_to_filer.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import filer.fields.image
+
+
+def migrate_to_filer(apps, schema_editor):
```

Comment:
I'd leave it like this as it is structured by:

- Migrating to filer (incl. addition / removal of related fields)
- Adapting models for new 2.x release